app/views/layouts/base.html.slim

```diff
@@ -14,5 +14,5 @@ html lang="en-US"
     = yield :styles
 
   body
-    = render "application/alerts"
+    = content_for?(:alerts) ? render(inline: '') : render('application/alerts')
```

Comment:
It does work, but your block looks cleaner and easier to read.

I'm actually trying to get it to render nothing if alerts are defined. If alerts are previously defined, they've already loaded on another part of the page.